index.bs

```diff
@@ -698,6 +731,8 @@ Note: the support of depth sensing API is not limited only to hardware classifie
 
 For each of {{XRDepthStateInit/depthTypeRequest}}, {{XRDepthStateInit/usagePreference}}, and {{XRDepthStateInit/dataFormatPreference}},  The device MUST have a <dfn>preferred [=native depth sensing=] capability</dfn> that MUST be used if the corresponding array is empty. The type, usage, and format SHOULD reflect the most efficient ones of the device, though they may be dependent upon each other.
 
+The device can be said to have a <dfn>depth sensing active state</dfn>, which is a boolean representing if depth sensing capabilities are actively running. This state should begin as <code>true</code>. If the native device cannot simulate this state, the user agent can simply not process any depth data when the state is otherwise <code>false</code>.
```

Comment:
should begin -> MUST begin

Reply:
@toji you all also essentially gave the same comment here ;).